### ISSUE TITLE
Django 1.9: cannot import name 'same_origin' 

### ISSUE THREAD

**Issue by shacker** [jbittel/django-mama-cas#22]
When running under Django 1.9 beta:

```
  File "/path/to/...lib/python3.5/site-packages/mama_cas/models.py", line 14, in <module>
    from django.utils.http import same_origin
ImportError: cannot import name 'same_origin'
```

Thanks.

**Comment by jbittel** [comment]
Ah, okay. I hadn't started testing under Django 1.9 yet but it doesn't look like this will be too hard to fix. Thanks for the reminder!

**Comment by jbittel** [comment]
Note that tests are still slightly broken under Django 1.9 because of the "url from future" import required by Django 1.4 in the warn template.

Thanks!

**Comment by cultavix** [comment]
Problem is still there. I'm using Django 1.9, installed using pip

<code>
ImportError: cannot import name same_origin
</code>

**Comment by jbittel** [comment]
This is fixed in 93b74f960b1db4007189a61804cea2089b6d9402, but is not yet pushed to PyPI. It will be included in the next release.
